companion/src/eeprominterface.cpp

```diff
@@ -1843,3 +1844,16 @@ SimulatorInterface *GetCurrentFirmwareSimulator()
   else
     return NULL;
 }
+
+void FlightModeData::setDefaultLinkedFlightModes(const int phase)
```

Comment:
As suggested I am in the process of modifying clear() to clear(const int i) and checking for calls.
Also found the radio model initialisation code section and just working thru the maze of compiler defs and macros to write the conditional initialisation of the rotary encoders.
Will update the pull request when ready for next peer review.